src/winetricks

```diff
@@ -6623,6 +6624,27 @@ load_directmusic()
 
 #----------------------------------------------------------------
 
+w_metadata directshow dlls \
+    title="DirectShow runtime DLLs (amstream, qasf, qcap, qdvd, qedit, quartz)" \
+    publisher="Microsoft" \
+    year="2011" \
+    media="download" \
+    file1="../win7sp1/windows6.1-KB976932-X86.exe"
+
+load_directshow()
+{
+    w_warn "You can specify individual DirectShow verbs instead. e.g. 'winetricks quartz'"
+
+    w_call quartz
+    w_call qasf
+    w_call qcap
+    w_call qdvd
+    w_call qedit
+    w_call amstream
```

Comment:
On mobile, so can't test, but do they need to be in this order?

Reply:
No, the order shouldn't matter as far as I'm aware.

Reply:
Could you sort them please? (Sorry, that should've been in my first comment)

Reply:
Yep, done.